ci-operator/step-registry/openshift/e2e/aws/cgroupsv1/openshift-e2e-aws-cgroupsv1-workflow.yaml

```diff
@@ -1,15 +1,19 @@
 workflow:
   as: openshift-e2e-aws-cgroupsv1
   steps:
+    allow_best_effort_post_steps: true
     pre:
     - chain: ipi-conf-aws
     - ref: openshift-manifests-cgroupsv1
+    - ref: ovn-conf
     - chain: ipi-install
     test:
     - ref: openshift-e2e-test
     post:
     - chain: gather-network
     - chain: gather-core-dump
     - chain: ipi-deprovision
+    env:
+      FAIL_ON_CORE_DUMP: "false"
```

Comment:
I don't see anything in the commit message explaining what this is about.  Can you provide more context?

Reply:
This workflow is taken from the[ existing workflow](https://github.com/openshift/release/blob/16dba6cf8a5b92a26c2bc1467f7a27a8d7b513ea/ci-operator/step-registry/openshift/e2e/aws/ovn/openshift-e2e-aws-ovn-workflow.yaml#L15) of `ovn` jobs. 

That was introduced with this PR - https://github.com/openshift/release/pull/20566